### DISCUSSION TITLE
a bug in import .tt file dialog

### DISCUSSION THREAD

**Discussion by peter1384** [BearWare/TeamTalk5#2335] [Ideas]
when pressing the import .tt file button, we are allowed to select any file, because of this. Files of type:  combo box  All Files (*)  collapsed  Alt+  t
but when pressing this button, this box should be set to *.tt
Thanks

**Comment by peter1384** [comment]
@CoBC I can fix this. Do you want to do it or should i do this?

**Comment by lbk2907** [comment]
My oppinion, if you can fix this, just make a pull request for that.
